src/components/pages/docs.js

```diff
@@ -114,8 +110,7 @@ function Docs({ initialState, PageTemplate }) {
   };
 
   const renderLanguage = documentLanguage => {
-    const lang = languageNameProvider.getData(language)[documentLanguage];
-    return <CountryFlagAndName code={lang.flag} name={`${documentLanguage} (${lang.name})`} flagOnly />;
+    return <LanguageFlagAndName language={documentLanguage} flagOnly />;
```

Comment:
Nice simplification 😎

Reply:
We had it in so many places that I felt it was time to extract it.